content/documentation/extensions/index.adoc

```diff
@@ -0,0 +1,167 @@
+---
+title: "Extensions"
+date: 2020-02-10T11:20:21+02:00
+draft: false
+type: "documentation"
+weight: 7
+---
+
+:linkattrs:
+:sectlinks:
+
+= Extensions
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Extensions
+:keywords: Kiali Documentation Extensions Addons
+:icons: font
+:imagesdir: /images/documentation/extensions
+
+== What is a Kiali Extension ?
+
+The Istio ecosystem is expanding and there is an increasing number of components providing additional functionality enriching Service Mesh technology.
+
+Additionally, Kiali project is growing with more features in the areas of Observability, Analysis, Validation and Management of Istio.
+
+An Extension is a mechanism and guideline to integrate third party components inside Kiali.
+
+== Examples of Kiali Extensions
+
+Imagine that you develop a new project to provide extra capabilities on top of Istio.
+
+Your project deploys extra components in the Istio Control Plane and typically the first way to interact with your project is using CLI.
+
+A Kiali Extension could be used to:
+
+- Manage specific configuration of your project in a separate section.
+- Enrich Kiali Actions adding new operations for your project.
+- Augment Kiali views to include specific results on graph, metrics or details.
+- Integrate your project in the Istio ecosystem using Kiali as first class User Interface dashboard.
+
+Kiali currently offers integration for 3scale API Management.
+
+In the following sections we will illustrate the guidelines using Kiali 3scale extension as an example.
```

Comment:
Again, just combine:
Kiali currently offers integration for 3scale API Management.   In the following sections we will illustrate the guidelines using the Kiali 3scale extension as an example.